config/manifests/bases/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     operators.operatorframework.io/project_layout: go
     repository: github.com/open-telemetry/opentelemetry-operator
     support: OpenTelemetry Community
-  name: opentelemetry-operator.vX.Y.Z
+  name: opentelemetry-operator.v0.0.0
```

Comment:
This also looks suspicious.

Reply:
the command `$(OPERATOR_SDK) generate kustomize manifests -q` causes it. 
I think it relates to the [PR#4276](https://github.com/operator-framework/operator-sdk/pull/4276) i mentioned.

Reply:
Got, then it does belong to the PR, right?

Reply:
added this change to the PR